Mage.Sets/src/mage/cards/z/ZinniaValleysVoice.java

```diff
@@ -0,0 +1,152 @@
+package mage.cards.z;
+
+import java.util.UUID;
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.common.SimpleStaticAbility;
+import mage.abilities.dynamicvalue.common.PermanentsOnBattlefieldCount;
+import mage.abilities.effects.ContinuousEffectImpl;
+import mage.abilities.effects.common.continuous.BoostSourceEffect;
+import mage.abilities.effects.common.continuous.GainAbilityControlledEffect;
+import mage.abilities.effects.common.continuous.GainAbilityControlledSpellsEffect;
+import mage.abilities.keyword.FlyingAbility;
+import mage.abilities.keyword.OffspringAbility;
+import mage.constants.*;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterCreatureCard;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.common.FilterNonlandCard;
+import mage.filter.predicate.mageobject.AnotherPredicate;
+import mage.filter.predicate.mageobject.BasePowerPredicate;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.game.Game;
+import mage.game.stack.Spell;
+import mage.players.Player;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import static mage.abilities.dynamicvalue.common.StaticValue.*;
+import static mage.constants.Duration.WhileOnBattlefield;
+
+/**
+ *
+ * @author DreamWaker
+ */
+public final class ZinniaValleysVoice extends CardImpl {
+
+    static final FilterCreaturePermanent bfilter = new FilterCreaturePermanent("other creatures with base power 1");
+
+    static {
+        bfilter.add(new BasePowerPredicate(ComparisonType.EQUAL_TO, 1));
+        bfilter.add(TargetController.YOU.getControllerPredicate());
+        bfilter.add(AnotherPredicate.instance);
+
+    }
+
+    static final PermanentsOnBattlefieldCount bcount = new PermanentsOnBattlefieldCount(bfilter);;
+
+    static final FilterNonlandCard cfilter = new FilterNonlandCard("creature spells");
+
+    static {
+        cfilter.add(CardType.CREATURE.getPredicate());
+        cfilter.add(AnotherPredicate.instance);
+    }
+
+    public ZinniaValleysVoice(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[] { CardType.CREATURE }, "{U}{R}{W}");
+
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.BIRD);
+        this.subtype.add(SubType.BARD);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(3);
+
+	    // Flying
+        this.addAbility(FlyingAbility.getInstance());
+
+        // Creature spells you cast have offspring 2.
+        this.addAbility(
+                new SimpleStaticAbility(new GainAbilityControlledSpellsEffect(new OffspringAbility("{2}"), cfilter)
+                        .setText("Creature spells you cast have offspring {2}.")));
+        this.addAbility(new SimpleStaticAbility(new GainAbilityControlledEffect(
```

Comment:
I don't like this workaround. Aside from it being a hack, I'm concerned that it might not comply with

> 702.175b. If a [spell](https://yawgatog.com/resources/magic-rules/#spell) has multiple instances of [offspring](https://yawgatog.com/resources/magic-rules/#offspring), each is paid separately and [triggers](https://yawgatog.com/resources/magic-rules/#trigger) based on the payments made for it, not any other instances of [offspring](https://yawgatog.com/resources/magic-rules/#offspring).

I will not accept this unless unit tests are added to demonstrate functionality in each case:
- Cast a spell and decline to pay offspring, no token
- Cast a spell and pay offspring, get token
- Cast a spell that already has offspring (some cost other than {2}), each of the four possibilities for paying offspring costs or not, correct number of tokens in each case

Reply:
I've tested the first two dozens of times, and it seems to work perfectly.
Just messed around with the last one, and sadly its not working right...

REALLY hope this wont end up as an "unable to be implemented" card!